data/primary.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-	"release_v_dts": "ai",
+	"release_v_o": "ai",
 	"stopped_type": "Muminus",
 	"release_v_stops": "p",
 	"primary_name": "DIOtail_75",
```

Comment:
start_mom is missing from this stanza, is that a problem?

Reply:
Are you referring to stop_mom?
stop_mom is not used as the variable at the moment.

start_mom is used though.